packages/maskbook/src/plugins/PoolTogether/SNSAdaptor/index.tsx

```diff
@@ -44,7 +46,11 @@ function Renderer(props: React.PropsWithChildren<{ url: string }>) {
     return (
         <MaskbookPluginWrapper pluginName="PoolTogether">
             <Suspense fallback={<SnackbarContent message="Mask is loading this plugin..." />}>
-                <PoolTogetherView />
+                <EthereumChainBoundary
+                    chainId={ChainId.Mainnet}
+                    isValidChainId={(chainId) => [ChainId.Mainnet, ChainId.Matic].includes(chainId)}>
```

Comment:
this field allow input array may be a better

```suggestion
                    isValidChainId={[ChainId.Mainnet, ChainId.Matic]}>
```

Reply:
A function parameter makes it able to express more complex logic.